classes/game.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './item'
+
+# Game class
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize
+    super(date, multiplayer, last_played_at)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
```

Comment:
@stephnna, 
the game class looks good so far but it's missing some things:

> the constructor of the game class should have the right associated parameters.
- The game class is inheriting from Item class, hence:

_def initialize(id, publish_date, archived, date, multiplayer, last_played_at)
    super(id, publish_date, archived)
    @multiplayer = multiplayer
    @last_played_at = last_played_at
end_